### PR TITLE
Fixes Issue #6187 - Visual Material Underline color should match placeholder color

### LINKED ISSUE #6187: Visual Material Entry underline color should reference PlaceholderColor
### Description
When using Visual Material Entry component, the underline color will be equivalent to TextColor.

### Steps to Reproduce

1. Create a Entry
2. Set Visual="Material"
3. Set PlaceholderColor to any color
4. Start app

### Expected Behavior
Following the Material Design guidelines, the underline's color should be the updated accordingly with the PlaceholderColor ([source](https://material.io/design/components/text-fields.html)).

### Actual Behavior
Currently, the underline's color is being set accordingly with the value provided for TextColor.

### Basic Information

- Version with issue: 3.6.0.344457

### PR DESCRIPTION
**Pull request by jgold6**
### Description of Change ###

- Replaces TextColor with Placeholdercolor for setting the underline color in the relavant Material renderers (or base classes if appropriate)

No automated UI test as I do not know how to check the underline color of an Entry using UITest.

### Issues Resolved ### 
- fixes #6187

### API Changes ###

 None

### Platforms Affected ### 

- iOS
- Android
- Tizen

### Behavioral/Visual Changes ###
- Underline color in a Material Entry will now match the PlaceholderColor rather than the TextColor

None

### Before/After Screenshots ### 

Unfixed_Screenshot
Fixed_Screenshot

### Testing Procedure ###

1. Create a Material Entry control and set the TextColor and the PlaceholderColor. 
2. Make sure the underline color in the Entry matches the PlaceholderColor

### PR Checklist ###

- [ ] Has automated tests
- [*] Rebased on top of the target branch at time of PR
- [*] Changes adhere to coding standard

### CODE REVIEW

File: Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6187.cs [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+			Content = layout;
+		}
+
+
+
+#if UITEST
+		[Test]
+		public void ImageShouldLayoutOnTopOfButton()
+		{
+
```
Comment (samhouts) [inline Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6187.cs]: No test?

Comment (jgold6) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6187.cs]: Yes, sorry, I did note that in the description: "No automated UI test as I do not know how to check the underline color of an Entry using UITest."

Comment (samhouts) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6187.cs]: We can probably just remove this code, then. Instead, we should add a label to the screen that tells the reviewer what to look for!

Comment (samhouts) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6187.cs]: Maybe also be a good candidate for the new tests suite that @hartez  is working on.

Comment (jgold6) [reply Xamarin.Forms.Controls.Issues/Xamarin.Forms.Controls.Issues.Shared/Issue6187.cs]: I added a label describing expected behavior and removed the UITest code. 

I have not heard about any new test suite, but would be happy to learn and try it for this PR.

**Pull request merged**